doc/llm.md

```diff
@@ -0,0 +1,27 @@
+---
+page:
+  image: llm.png
```

Comment:
Shouldn't you also actually show this in the doc itself?

I don't see it in rendered page:

<img width="1255" alt="image" src="https://github.com/juspay/services-flake/assets/3998/50ed6936-2ecd-491b-a455-b42270d318f1">

Reply:
I am not sure where to put it

Reply:
Okay, I'll take a look.